### ISSUE TITLE
Why logits includes reduction of the number of neurons?

### ISSUE THREAD

**Issue by maayange17** [loeweX/Forward-Forward#9]
<img width="486" height="95" alt="Image" src="https://github.com/user-attachments/assets/7157be0b-1067-43a6-9077-55c82d033136" />

**Comment by whubaichuan** [comment]
@maayange17 I think here,`Z.shape[1]` referres to the threshold, i.e., the theta (eq.1) in [FF](https://arxiv.org/pdf/2212.13345) paper. Also, you can look [this paper](https://whubaichuan.github.io/data/BinaryFO.pdf)'s section II-B for easiler understanding.
